Bio/SeqIO/QualityIO.py

```diff
@@ -1048,6 +1101,101 @@ def FastqPhredIterator(handle, alphabet=single_letter_alphabet, title2ids=None):
         yield record
 
 
+def FastqFourLineIterator(handle, alphabet=single_letter_alphabet, title2ids=None):
```

Comment:
Probably we can refactor this to avoid the code duplication, any ideas?

Reply:
I couldn't think of anything, sorry! I had a doubt here, actually - do we still want to allow the use of the general iterator or do we simply replace it by the stricter one?